src/share/classes/com/sun/javatest/regtest/util/ProcessUtils.java

```diff
@@ -27,20 +27,35 @@
 
 import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
+import java.util.Objects;
 
 /**
  * Utilities for handling Processes.
  */
 public class ProcessUtils {
 
-    private static Method destroyForciblyMethod;
+    private static final Method destroyForciblyMethod;
+    private static final Method PID_METHOD;
```

Comment:
How much we we still need reflective access to these methods?  
If they are only invoked from the `jtreg` tool (and never from a test VM), then the minimum level of JDK required to run `jtreg` is currently 11, right? And, we no longer support running test-VM code on anything less than 8, right?

So, at some level, it seems we may not need `ProcessUtils` any more -- unless maybe as a convenience wrapper.

Reply:
Hello Jon,

> And, we no longer support running test-VM code on anything less than 8, right?

I think you are right. Although, the `Process.pid()` is only available in Java 9+, I think that call is only run on the `jtreg` tool's VM which as you note seems to require Java 11 (I will have to verify some JDK CI runs to be sure that's the case currently). 

If it's OK, then I'll take up the usage and possible remove of this reflection code as a separate issue and verify things work fine without it.

Reply:
I created [CODETOOLS-7903789](https://bugs.openjdk.org/browse/CODETOOLS-7903789) follow-up issue to replace those reflection-based method invocations. Let's keep the scope of changes of the PR small(er), in order to release jtreg 7.5 soon(er).